scripts/quickstart.sh

```diff
@@ -467,7 +473,7 @@ function run {
       if [ $PLATFORM == "LINUX" ]; then
         cpuCores=$(nproc --all)
       fi
-      NUM_RPC_THREADS=cpuCores
+      NUM_RPC_THREADS=$cpuCores
       NUM_HTTP_THREADS=$(( 4*cpuCores ))
```

Comment:
@Andrew-Pohl need o add this here too :)

Reply:
The arthmetic expression should mean you don't have to there. (They are both technically right but not needed)

Reply:
great, was making sure you didn't miss it